src/pages/iou/steps/MoneyRequstParticipantsPage/MoneyRequestParticipantsPage.js

```diff
@@ -104,6 +107,7 @@ function MoneyRequestParticipantsPage(props) {
                             onAddParticipants={IOU.setMoneyRequestParticipants}
```

Comment:
I can't comment on the line above for the title in the header, but this will fix Bug9

```
title={isDistanceRequest ? props.translate('common.distance') : props.translate('iou.cash')}
```

Reply:
will commit this

Reply:
committed